sympy/concrete/summations.py

```diff
@@ -429,6 +433,15 @@ def is_convergent(self):
 
         order = O(sequence_term, (sym, S.Infinity))
 
+        ### ----------- ratio test ---------------- ###
+        next_sequence_term = sequence_term.xreplace({sym: sym + 1})
+        ratio = simplify(next_sequence_term/sequence_term)
+        lim_ratio = limit(ratio, sym, upper_limit)
+        if (lim_ratio) > 1:
+            return S.false
+        if (lim_ratio) < 1:
```

Comment:
Is this test for series with positive terms only? Consider something like `sequence_term = (-1)**n`.

Reply:
`sequence_term = (-1)**n` will be caught by the 'Divergence test` code snippet.
And yes, this is intended for positive terms only.

Reply:
There should probably be a comment explaining the assumptions for this test for future maintenance.

Reply:
I''ll add a comment about the test.
Are there any problems in the other changes?

Reply:
For a concrete example, here is a wrong result we get because of incorrect ratio test implementation.
```
>>> Sum((-2)**n + (-3)**n, (n, 1, oo)).is_convergent()
True
```